docs/content/en/docs/packages/cluster-autoscaler/addclauto.md

```diff
@@ -12,7 +12,7 @@ Be sure to refer to the [troubleshooting guide]({{< relref "../troubleshoot" >}}
 
   {{% alert title="Important" color="warning" %}}
    * Starting at `eksctl anywhere` version `v0.12.0`, packages on workload clusters are remotely managed by the management cluster.
-   * While following this guide to install packages on a workload cluster, please make sure the `kubeconfig` is pointing to the management cluster that was used to create the workload cluster. The only exception is the `kubectl create namespace` command below, which should be run with `kubeconfig` pointing to the workload cluster.
+   * While following this guide to install packages on a workload cluster, please make sure the `kubeconfig` is pointing to the management cluster that was used to create the workload cluster.
```

Comment:
Can we:
- undo this change. it is relevant to this part of the instructions: https://github.com/aws/eks-anywhere/blob/1e81579e0d8d8d997b1350985786fd24cbaa3109/docs/content/en/docs/packages/cluster-autoscaler/addclauto.md#L131-L131
- verify that we document the package's controller's ability to manage packages on workload clusters and then kill the first bullet in this note
- move the remainder of this note to below this header: https://anywhere.eks.amazonaws.com/docs/packages/cluster-autoscaler/addclauto/#install-cluster-autoscaler-in-workload-cluster

Reply:
Would it be better to reword some of this instead then?

```
The only exception is when you need to create a new namespace. The command `kubectl create <namespace>` should be executed with `kubeconfig` pointing to the workload cluster
```

I was confused a little bit going through the instructions because the current wording makes it sound like there is a step in the instructions where a namespace is created, but it's not actually referenced on the page.

Reply:
Updated. I did find that the package's controller ability to manage packages on workload clusters is mentioned [here](https://github.com/aws/eks-anywhere/blob/0fc695e37988bb8bdfe4a2aeb5867784117cfa14/docs/content/en/docs/packages/overview.md?plain=1#L15) in the overview.